validate/error.go

```diff
@@ -0,0 +1,69 @@
+package validate
```

Comment:
The runtime-tool and image-tool are separated, where should we have such a 'stand-alone module', "opencontainers/certification"?